src/ontology/cl-edit.owl

```diff
@@ -32019,6 +32024,50 @@ AnnotationAssertion(rdfs:label obo:CL_4042006 "dural macrophage"@en)
 SubClassOf(obo:CL_4042006 obo:CL_4042003)
 SubClassOf(obo:CL_4042006 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002092))
 
+# Class: obo:CL_4042007 (protoplasmic astrocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:25904839") Annotation(oboInOwl:hasDbXref "PMID:34616062") Annotation(oboInOwl:hasDbXref "PMID:37139179") Annotation(oboInOwl:hasDbXref "PMID:37824655") obo:IAO_0000115 obo:CL_4042007 "An astrocyte cell that has highly branched protrusions, its soma is part of a neocortex and it is found between layer 2 and 6. A protoplasmic astrocyte is involved in the formation and elimination of synapses, in the clearance of glutamate, in the modulation of synaptic functions and in the regulation of blood flow in response to synaptic activity.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042007 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4042007 "2024-04-08T16:01:24Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:20012068") rdfs:comment obo:CL_4042007 "This astrocyte extends its branches to several hundred of dendrites to multiple neurons to envelope 100,000 or more synapses. In humans, a protoplasmic astrocyte has an average diameter of 142 µm, and its diameter ranges between 100 and 400 µm. In rodents the average diameter of a protoplasmic astrocites is 56 µm.")
```

Comment:
Also wrong AP? Text showing up as on dbxref rather than comment in diff.

Reply:
It is fixed now